src/components/pages/CaseOverview/CaseDetails.js

```diff
@@ -61,6 +61,23 @@ const CaseDetails = props => {
         <p>{`Country of Origin: ${caseData.country_of_origin}`}</p>
         <p>{`Filed Within One Year: ${caseData.filed_in_one_year}`}</p>
         <p>{`Applicant Language: ${caseData.applicant_language}`}</p>
+        {/* the segment below displays the disclaimer message with the details requested by the stakeholders */}
```

Comment:
Since some of the data below this point is repeated (albeit in a different format), this comment helps other developers understand why this segment was added to begin with.